anyway/views/news_flash/api.py

```diff
@@ -194,16 +194,14 @@ def gen_news_flash_query_v2(session, valid_params: dict):
             sources = [source.value for source in value]
             query = query.filter(NewsFlash.source.in_(sources))
         if param == "start_date":
-            query = query.filter(
-                NewsFlash.date >= value and NewsFlash.date <= valid_params["end_date"]
-            )
+            query = query.filter(value <= NewsFlash.date <= valid_params["end_date"])
         if param == "resolution":
             query = filter_by_resolutions(query, value)
     query = query.filter(
         and_(
             NewsFlash.accident == True,
             not_(and_(NewsFlash.lat == 0, NewsFlash.lon == 0)),
-            not_(and_(NewsFlash.lat == None, NewsFlash.lon == None)),
+            not_(and_(NewsFlash.lat is None, NewsFlash.lon is None)),
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969